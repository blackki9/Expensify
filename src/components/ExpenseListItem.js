import React from 'react';

const ExpenseListItem = ({description, amount, createdAt}) => (
    <div>
     {description} {amount} {createdAt}
    </div>
);


export default ExpenseListItem;