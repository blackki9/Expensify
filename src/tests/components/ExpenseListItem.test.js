import React from 'react';
import {shallow} from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('should render expense item with data', () => {
    const wrapper = shallow(<ExpenseListItem expense={{...expenses[0]}} />);
    expect(wrapper).toMatchSnapshot();
});