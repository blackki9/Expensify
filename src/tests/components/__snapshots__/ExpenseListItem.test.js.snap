// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense item with data 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/undefined"
  />
   
</div>
`;
